cuda/test/matrix/sellp_kernels.cpp

```diff
@@ -261,7 +239,8 @@ TEST_F(Sellp,
 
 TEST_F(Sellp, ApplyToComplexIsEquivalentToRef)
 {
-    set_up_apply_matrix();
+    set_up_apply_matrix(64);
+
     auto complex_b = gen_mtx<ComplexVec>(231, 3);
```

Comment:
Including this empty line (and the one in the next test) since the setup step is supposed to be part of the Allocation block:
```suggestion
    set_up_apply_matrix(64);
    auto complex_b = gen_mtx<ComplexVec>(231, 3);
```